docs/pages/blog/material-ui-next-summer-2025.md

```diff
@@ -0,0 +1,136 @@
+
+<div style="max-width:80%; margin:0 auto;">
+
+| Package        | Status                           | Support scope        |
+| -------------- | -------------------------------- | -------------------- |
+| Material UI v7 | Long term support until 2028     | bug & security fixes |
+| Material UI v6 | Long term support until 2028     | bug & security fixes |
+| new-lib        | pre-alpha - release by Sept 2026 | active development   |
+
+</div>
+
+## Migration and continuity
+
+We understand that adopting a new major can be intimidating. That’s why we’re committed to making the transition as smooth as possible. When new-lib reaches beta later this year, you can expect:
+
+**An incremental migration path.** You’ll be able to install new-lib alongside Material UI and gradually migrate screens or components. We’ll publish guides, codemods, AI tooling, and examples to help you update your code.
+
+**Long-term support for prior majors.** Material UI will continue receiving critical fixes while you upgrade, similar to how MUI X extended the LTS window when v8 was released.
+
+**Dedicated support options.** Our Customer Success team is scaling up to provide tailored assistance to our enterprise customers. Whether you need training, hands-on migration help, or architectural guidance, we’ll be there.
+
+## Material Design 3: your vote matters
+
+We acknowledge that Material Design may still have a place in the ecosystem, but we also recognise that it shouldn’t be the default for every product. To decide where to prioritize our resources, we’re asking the community to vote on whether Material Design 3 should be one of the first official themes for new-lib.
+
+Have your say →: [Community Poll (1min)](https://tally.so/r/w8X8Po) 🗳️
+
+## What's next
+
+Our roadmap for new-lib is ambitious. Over the coming months, we’ll release RFCs, beta packages, and developer previews (starting at the end of 2025). In parallel, we’ll continue to deliver improvements to the existing Material UI ecosystem, including updates and new features for our advanced components scheduled for March 2026. Here are a few initiatives on our radar:
+
+**Additional themes.** Beyond a default white-label theme, we plan to ship themes inspired by other design systems and brand palettes. Material Design 3 will be a theme depending on the community's wishes.
+
+**AI-ready components.** Just as MUI X is exploring AI-assisted features, we’re experimenting with patterns and APIs that make AI-driven UIs straightforward to build.
+
+**Stable release around September/October 2026,** though the exact timeline is still subject to change based on your feedback and the results of our beta program.
+
+## How to get involved
+
+Your feedback will shape all of these initiatives. We can’t wait to hear what you think and to build the next generation of UI components together.
+
+- **Join the discussion:** [GitHub Discussions](https://github.com/mui/material-ui/discussions)
```

Comment:
The idea here is to create a discussion (just before releaseing the post) to host community's comments.